themes/learn.jquery.com/content-page.php

```diff
@@ -20,7 +20,7 @@
 		<div class="bottom-links row">
 			<?php if (isset($next_prev_arr['prev'])): ?>
 				<div class="prev six columns">
-					<a href="<?php echo $next_prev_arr['prev']->guid; ?>">
+					<a href="<?php echo get_page_link( $next_prev_arr['prev']->ID ); ?>">
```

Comment:
Could you remove these changes? We already had an open PR for that: https://github.com/jquery/jquery-wp-content/pull/358. I feel like we should attribute @sigorilla in here as [he was the one that fixed it](https://github.com/jquery/learn.jquery.com/issues/647#issuecomment-90430039).